functions/Enums.js

```diff
@@ -10,24 +10,24 @@
 
 // TODO: temporary for pilot, manually setting a fixed umbrella
 exports.UmbrellaId = {
-    TEST: 'PrE7oCZajNcRAOXNAFwRQizQQcJ3'
-    // TODO: create a real umbrella for UW
+    TEST: 'NLRZo1xfPHOI8rGpzSKiEJVOOUt2',
+    UW: 'bfwL94u7XYZyhZpDByXLeWb5ivA3',
```

Comment:
**ignore** changes in this file, just syncing `functions/Enums.js` with `src/Enums.js`